queries.sql

```diff
@@ -100,3 +100,53 @@ ON animals.owner_id = owners.id
 GROUP BY owners.full_name
 ORDER BY COUNT(name) DESC
 LIMIT 1
+
+ALTER TABLE visits
+ADD COLUMN date_of_visit date
+
+SELECT name,animal_id,vet_id,date_of_visit FROM visits JOIN animals
+ON animal_id = id
+WHERE vet_id = 1 AND date_of_visit = (SELECT MIN(date_of_visit) FROM visits WHERE vet_id = 1);
+
+SELECT name,animal_id,vet_id FROM visits JOIN animals
+ON animal_id = id
+WHERE vet_id = 3 
+
+SELECT vets.name,species.name FROM species
+ FULL JOIN specializations
+  ON species.id = specializations.species_id
+ FULL JOIN vets
+  ON specializations.species_id = vets.id;
```

Comment:
- [ ] Please add labels too to your screenshots in the pull request, and I would advise you to use a tool such as the **_Windows snipping tool_** - [(click me to learn)](https://support.microsoft.com/en-us/windows/open-snipping-tool-and-take-a-screenshot-a35ac9ff-4a58-24c9-3253-f12bac9f9d44) to crop the portion of the screen that has only the code. Like so 👇🏾 

   #### Who was the last animal seen by William Tatcher?
![image](https://user-images.githubusercontent.com/84156714/236463400-ee0b6935-1873-47de-b884-83f70cf66bde.png)

   #### How many different animals did Stephanie Mendez see?
![image](https://user-images.githubusercontent.com/84156714/236463400-ee0b6935-1873-47de-b884-83f70cf66bde.png)